codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/EnumGenerator.kt

```diff
@@ -57,11 +56,17 @@ internal class EnumMemberModel(private val definition: EnumDefinition, private v
 }
 
 private fun RustWriter.docWithNote(doc: String?, note: String?) {
-    doc?.also { docs(escape(it)) }
-    note?.also {
-        // Add a blank line between the docs and the note to visually differentiate
-        doc?.also { write("///") }
-        docs("**NOTE:** $it")
+    if (doc.isNullOrBlank() && note.isNullOrBlank()) {
+        // If the model doesn't have any documentation for the shape, then suppress the missing docs lint
+        // since the lack of documentation is a modeling issue rather than a codegen issue.
+        rust("##[allow(missing_docs)] // documentation missing in model")
+    } else {
+        doc?.also { docs(escape(it)) }
+        note?.also {
+            // Add a blank line between the docs and the note to visually differentiate
+            doc?.also { write("///") }
+            docs("**NOTE:** $it")
```

Comment:
Should this use the same note format as everything else (with `# Note` as a standalone heading)?